src/main/scala/nl.knaw.dans.easy.dd2d/Inbox.scala

```diff
@@ -42,7 +42,12 @@ class Inbox(dir: File, dansBagValidator: DansBagValidator, dataverse: DataverseI
     dirs.foreach {
       d => {
         debug(s"Adding $d")
-        q.add(DepositIngestTask(Deposit(d), dansBagValidator, dataverse, publish = autoPublish))
+        try {
+          q.add(DepositIngestTask(Deposit(d), dansBagValidator, dataverse, publish = autoPublish))
+        }
+        catch {
```

Comment:
Catching all objects of type `Exception` is a bad practice. If you look at what this includes, there are a lot of things that I would not like to be caught at all. **For example** NullPointerException, which is virtually always just a sign of sloppy programming. 

![exception hierarchy](https://www.benchresources.net/wp-content/uploads/2017/02/exception-hierarchy-in-java.png)